lightning-tag/implementation-google-ad-manager.md

```diff
@@ -33,35 +33,41 @@ Before using this snippet, please replace the instances of `<lotameClientId>` wi
   <script>
     ! function() {
       window.googletag = window.googletag || {};
-      window.googletag.cmd = window.googletag.cmd || []; 
+      window.googletag.cmd = window.googletag.cmd || [];
+      var targetingKey = 'lotame';
+      var lotameClientId = '<lotameClientId>';
+      var audLocalStorageKey = 'lotame_' + lotameClientId + '_auds';
 
       // Immediately get audiences from local storage and get them loaded
       try {
-        var localStorageAudiences = window.localStorage.getItem('lotame_<lotameClientId>_auds') || '';
-        googletag.cmd.push(function() {
-          window.googletag.pubads().setTargeting('lotame', localStorageAudiences);
-        });  
+        var storedAuds = window.localStorage.getItem(audLocalStorageKey) || '';
+
+        if (storedAuds) {
+          googletag.cmd.push(function() {
+            window.googletag.pubads().setTargeting(targetingKey, storedAuds.split(','));
+          });
+        }
       } catch(e) {
       } 
 
       // Callback when targeting audience is ready to push latest audience data
       var audienceReadyCallback = function (profile) {
 
-        // Get audiences in a comma-separated string which conforms to Google Ads input format
-        var lotameAudiences = profile.getAudienceString(',') || '';
+        // Get audiences as an array
+        var lotameAudiences = profile.getAudiences() || [];
```

Comment:
we got rid of getAudiences() on the data object, but profiles still has one i think so this is OK i think, just recognized the name as one we got rid of so annotating

Reply:
Correct - this is not the one being dropped. I will add the new getAudiences() on the namespace to the docs once we change it.